src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs

```diff
@@ -24,6 +24,9 @@ public override void Configure(IFunctionsHostBuilder builder)
 #else
             builder.Services.AddScoped<IRepository<Resume>, ResumesInMemoryRepository>();
 #endif
+            builder.Services.AddScoped<ISkillsService, SkillsService>();
+            builder.Services.AddScoped<ISkillsRepository, SkillsInMemoryRepository>();
```

Comment:
since you have implemented the API integration, please remove this line, in memory won't be used